src/js/dom.js

```diff
@@ -0,0 +1,246 @@
+'use strict';
+
+function dq(sel, ctx) {
```

Comment:
please use speaking variable names and add jsdoc everywhere. I would really appreciate this, thank you.